docker/run_on_entry.sh

```diff
@@ -28,17 +28,21 @@ sed -i "s|\['allowed_for_get'\]|$ALLOWED_FOR_GET|g" /var/www/html/apel_rest/sett
 
 # fetch the crl first
 fetch-crl
-# then start the periodic fetch-url
-service fetch-crl-cron start
+
+# alter the fetch-crl cron to run regardless of any services
+echo "# Cron job running by default every 6 hours, at 45 minutes past the hour
+# with  +/- 3 minutes sleep.
+
+45 */6 * * * root /usr/sbin/fetch-crl -q -r 360" >  /etc/cron.d/fetch-crl
```

Comment:
Just checking that you want a pipe and not an append here. What would be in the fetch-crl cron beforehand?

Reply:
yes, it looks like this

```
# Cron job running by default every 6 hours, at 45 minutes past the hour
# with  +/- 3 minutes sleep.
# The cron job file can be enabled/disabled with
# systemctl start fetch-crl-cron
# systemctl enable fetch-crl-cron

45 */6 * * * root ! /bin/systemctl -q is-active fetch-crl-cron.service || /usr/sbin/fetch-crl -q -r 360
```

So this change just removes the check that the dummy service is running

Reply:
ok.